code-challenges.test.js

```diff
@@ -27,25 +36,100 @@ const colors2 = [
 ]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
 
+describe("color_Filter", () => {
+  it("returns an array with the first item remove, then shuffles the array", () => {
+    expect(["purple", "blue", "green", "yellow", "pink"]).toEqual(expect.arrayContaining(colors1))
+})
+})
+
```

Comment:
To save lines when writing code you can combine the tests to expect two things! For example:
describe("color_Filter", () => {
  it("returns an array with the first item remove, then shuffles the array", () => {
    expect(["purple", "blue", "green", "yellow", "pink"]).toEqual(expect.arrayContaining(colors1))
  
expect(colors2).toEqual(expect.arrayContaining(colors2))  
    
})
})

This will ensure DRY code is written.